### ISSUE TITLE
Seperate out RxJavaAssemblyTracking

### ISSUE THREAD

**Issue by timmutton-agl** [akarnokd/RxJava2Extensions#39]
I'd like to include RxJavaAssemblyTracking in a project, but don't need the other stuff. would it be possible for RxJavaExtensions to be its own lib?

**Comment by akarnokd** [comment]
No. It would be a maintenance burden. Also why is it a problem, Android? ProGuard should work.
